examples/autoqasm/1_Getting_started_with_AutoQASM.ipynb

```diff
@@ -108,7 +108,7 @@
    "source": [
     "device = LocalSimulator()\n",
     "result = device.run(bell_state, shots=100).result()\n",
-    "counts = Counter(result.measurements[\"c\"])\n",
+    "counts = Counter(result.measurements[\"return_value\"])\n",
```

Comment:
Once we implement outputs in the result object, will this help the user avoid having to use magic string literals like `"return_value"` (or `"return_value0"`, etc. in the case of tuples)?

Reply:
Yes, exactly. That's the piece that we can't do without service changes. I can hack in something that allows us to use `result.output` instead of `result.measurements["return_value"]`, but it breaks down when the user returns a named variable, like we have in the IPE demo. I think it's probably best to wait until we can do the complete solution, rather than a hack, but that could be a discussion question for Monday

Reply:
For multiple returned values, it would be `outputs[0]`, `outputs[1]`